test/commands/environments/destroy.test.ts

```diff
@@ -3,23 +3,93 @@ import PipelineUtils from '../../../src/architect/pipeline/pipeline.utils';
 import { mockArchitectAuth, MOCK_API_HOST } from '../../utils/mocks';
 
 describe('environment:destroy', () => {
-
   // set to true while working on tests for easier debugging; otherwise oclif/test eats the stdout/stderr
   const print = false;
 
   const mock_account = {
     id: 'test-account-id',
-    name: 'test-account'
-  }
+    name: 'test-account',
```

Comment:
Could you add tests for a strict `environment:create` command as well please? I see that feature is already merged, but I don't couldn't find any tests for it

Reply:
Added environment:create tests as well